packages/fxa-shared/payments/stripe-firestore.ts

```diff
@@ -116,6 +131,9 @@ export class StripeFirestore {
 
     const uid = customer.metadata.userid;
     if (!uid) {
+      if (ignoreErrors) {
+        return customer;
+      }
```

Comment:
Because we want to ignore Firestore caching errors. If we throw an error, than upstream code not only has to ignore them, it has to fetch the customer from Stripe again directly.

Reply:
But this error isn't just a caching error -- we would have other problems if the Stripe customer has no uid, right? We wouldn't be able to associate that customer and their subscriptions with an FxA account.